google/src/main/scala/org/broadinstitute/dsde/rawls/google/GoogleUtilities.scala

```diff
@@ -46,6 +46,10 @@ trait GoogleUtilities extends LazyLogging with InstrumentedRetry with GoogleInst
     }
   }
 
+  protected def executeGoogleRqWithRetry[T](request: AbstractGoogleClientRequest[T])(implicit counters: GoogleCounters, histo: Histogram): Future[T] = {
```

Comment:
spell out request

Reply:
what's the country of origin? can you use it in a sentence?

:bee: